src/components/Header/Header.tsx

```diff
@@ -0,0 +1,73 @@
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos'
+import { useHistory } from 'react-router'
+import styled from 'styled-components'
+import Button from '../common/Button/Button'
+
+interface Props {
+  withBack?: Boolean
+  children: string
+}
+
+function Header({ withBack = false, children }: Props): JSX.Element {
+  const { goBack } = useHistory()
```

Comment:
I don't really know typscript, but while reading your code, i ask me why here is no prop definition?